src/sp/it/pl/util/system/Environment.kt

```diff
@@ -301,7 +289,9 @@ fun saveFile(title: String, initial: File? = null, initialName: String, w: Windo
     return if (f!=null) Try.ok(f) else Try.error()
 }
 
-/** @return file representing the currently used wallpaper or null if error not supported */
+/** @return file representing the current desktop wallpaper or null if not supported
+ *
+ * Currently only supported on Windows */
```

Comment:
Implementation detail. The documentation already says that it may not be supported.